src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py

```diff
@@ -0,0 +1,120 @@
+from swsscommon import swsscommon
+
+from .log import log_err, log_info, log_crit
+from .manager import Manager
+from .utils import run_command
+
+
+class BBRMgr(Manager):
+    """ This class initialize "BBR" feature for  """
+    def __init__(self, common_objs, db, table):
+        """
+        Initialize the object
+        :param common_objs: common object dictionary
+        :param db: name of the db
+        :param table: name of the table in the db
+        """
+        super(BBRMgr, self).__init__(
+            common_objs,
+            [("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME, "localhost/bgp_asn"),],
+            db,
+            table,
+        )
+        self.enabled = False
+        self.bbr_enabled_pgs = {}
+        self.directory.put(self.db_name, self.table_name, 'status', "disabled")
+        self.__init()
+
+    def set_handler(self, key, data):
+        """ Implementation of 'SET' command for this class """
+        if not self.enabled:
+            log_info("BBRMgr::BBR is disabled. Drop the request")
+            return True
+        if not self.__set_validation(key, data):
+            return True
+        cmds = self.__set_prepare_config(data['status'])
+        rv = self.cfg_mgr.push_list(cmds)
+        if not rv:
+            log_crit("BBRMgr::can't apply configuration")
+            return True
```

Comment:
always return True?

Reply:
Yes. False means save it in the queue, and give it back to me next time,
True means - remove it completely.
In this case I don't see any reason the change could be applied without any issue in the future